packages/math-inline/src/main.jsx

```diff
@@ -61,6 +61,16 @@ function prepareForStatic(model, state) {
   }
 }
 
+// increase the font of the parallel notation
+const updateSpans = () => {
+  const spans = Array.from(document.querySelectorAll('span[mathquill-command-id]'));
```

Comment:
Can we use the function that was defined in pie-lib in this PR: https://github.com/pie-framework/pie-lib/pull/343?

Reply:
@PatriciaRomaniuc we have pie-lib released